iOS/ViewControllers/Courses/PeerAssessmentViewController.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  Created for xikolo-ios under MIT license.
+//  Copyright © HPI. All rights reserved.
+//
+
+import Common
+import UIKit
+
+class PeerAssessmentViewController: UIViewController {
+
+    private static let pointsFormatter: NumberFormatter = {
+        let formatter = NumberFormatter()
+        formatter.decimalSeparator = "."
+        formatter.minimumFractionDigits = 1
+        formatter.maximumFractionDigits = 1
+        return formatter
+    }()
+
+    private static let dateFormatter: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "dd. MMM yyyy"
+        return formatter
+    }()
```

Comment:
Unfortunately, this date format is not localized. Additionally, this does not include the time and time zone, which are important for the learner when dealing with deadlines ;) Have a look at `\CourseData.formattedDateWithTimeZone`. You can reuse this code to achieve a uniform deadline format in the app.